blocks/init/src/Blocks/components/copyright/components/copyright-options.js

```diff
@@ -12,7 +12,7 @@ export const CopyrightOptions = (attributes) => {
 
 	return (
 		<Toggle
-			label={__('Copyright', 'eightshift-frontend-libs')}
+			label={__('Copyright', '%g_textdomain%')}
```

Comment:
I'm probably missing something but how come this `textdomain` convention is only used in this and `site-footer-options.js` files?

Reply:
@piqusy what? 😄 
![image](https://github.com/infinum/eightshift-frontend-libs/assets/135589039/ead20bef-8e67-4eca-b664-07d6fdb77a18)

Reply:
this was all done for the other blocks in the previous PR so I fixed the additional changes Kanc did for this block

Reply:
Oh, it seems that I've clicked on a link for a single commit and saw just that small change 😄